_posts/2021-03-23-solidity-0.8.3-release-announcement.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.3 Release Announcement'
+date: '2021-03-23'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.3](https://github.com/ethereum/solidity/releases/tag/v0.8.3) fixes the [Solidity Optimizer Keccak Caching Bug](https://blog.soliditylang.org/2021/03/23/keccak-optimizer-bug/), which is present in all prior versions of Solidity, and includes two changes to the optimizer (improve storage writes of small types and replace multiplications and divisions with powers of two by shifts).
+
```

Comment:
We could mention that the improvements should provide a big gas benefit when writing structs that cover a full storage slot to storage.

Reply:
added